src/actions/flashMessageActions.js

```diff
@@ -1,23 +1,64 @@
 'use strict';
 
-import * as types from './actionTypes';
+import Noty from 'noty';
 
-export function flashRemove(flashMessage) {
-  return { type: types.FLASH_REMOVE, flashMessage };
-}
+// messageType affects the visual markup to distinguish severity
+export const messageType = {
+  ERROR: 'error',
+  INFO: 'info',
+  SUCCESS: 'success',
+  WARNING: 'warning',
+};
+
+// Display duration for a message.
+// bool false means forever.
+// integers are a duration in milliseconds.
+export const messageTTL = {
+  FOREVER: false,
+  SHORT: 1500,
+  MEDIUM: 3000,
+  LONG: 10000,
+  VERYLONG: 30000,
+};
 
-export function flashAdd(flashMessage) {
-  return function(dispatch) {
-    dispatch({ type: types.FLASH_ADD, flashMessage });
+export class FlashMessage {
+  constructor(text, type, ttl, subtext) {
+    this.text = text;
+    if (subtext) {
+      this.text = '<p>' + text + '</p><p>' + subtext + '</p>';
```

Comment:
```suggestion
      this.text =  '<b>' + text + '</b>' + '<br/>' + subtext;
```

Reply:
I noticed that the flash messages with subtext looked a bit different. Before:
<img width="366" alt="screenshot 2019-03-06 at 14 54 07" src="https://user-images.githubusercontent.com/455309/53861915-9554bb00-4020-11e9-8ab6-3412e6ef5c08.png">

after:
<img width="369" alt="screenshot 2019-03-06 at 14 55 16" src="https://user-images.githubusercontent.com/455309/53861918-984fab80-4020-11e9-8424-aac2342760e1.png">